### ISSUE TITLE
fails to start

### ISSUE THREAD

**Issue by miguelaht** [X3eRo0/dired.nvim#2]
I'm getting this error when executing ```:Dired``` or ```nvim .```.

```
E5108: Error executing lua .../nvim/site/pack/packer/start/dired.nvim/lua/dired/fs.lua:235: attempt to index a nil value
stack traceback:
        .../nvim/site/pack/packer/start/dired.nvim/lua/dired/fs.lua:235: in function 'New'
        ...vim/site/pack/packer/start/dired.nvim/lua/dired/dirs.lua:15: in function 'get_dir_content'
        .../site/pack/packer/start/dired.nvim/lua/dired/display.lua:56: in function 'get_dired_listing'
        .../site/pack/packer/start/dired.nvim/lua/dired/display.lua:79: in function 'display_dired_listing'
        .../site/pack/packer/start/dired.nvim/lua/dired/display.lua:37: in function 'render'
        ...im/site/pack/packer/start/dired.nvim/lua/dired/dired.lua:58: in function 'init_dired'
        ...im/site/pack/packer/start/dired.nvim/lua/dired/dired.lua:82: in function 'open'
        [string ":lua"]:1: in main chunk
```

Os: MacOs
Nvim version: v0.8.0-dev+595-g95c65a6b2

**Comment by X3eRo0** [comment]
Hey, @miguelaht can you check if the issue is fixed in [5d5a969](https://github.com/X3eRo0/dired.nvim/commit/5d5a969a55d6351a8de99aa03c2bfa71696f6387)

**Comment by miguelaht** [comment]
yes, it works now. Thanks.
